src/crepe/api/Force.cpp

```diff
@@ -0,0 +1,14 @@
+#include "Force.h"
+#include <cmath>
+
+namespace crepe::api {
+
+Force::Force(uint32_t gameObjectId, uint32_t forceMagnitude, uint32_t direction): Component(gameObjectId)
+{ 
+	// Convert direction from degrees to radians
+    float radian_direction = static_cast<float>(direction) * (M_PI / 180.0f);
```

Comment:
Er moet ergens vastgesteld worden dat overal in de engine `direction` of `angle` een hoek in radialen betekent, en alleen conversiefuncties de specifieke naam van de soort hoek geven (d.w.z. `deg_to_rad()` en `rad_to_deg()`)